src/ThaiRomanizationSharp.Thai2Rom/NeuralNet/Seq2Seq.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Diagnostics;
+using System.Linq;
+using TorchSharp;
+using static TorchSharp.torch;
+
+namespace ThaiRomanizationSharp.Thai2Rom.NeuralNet
+{
+    using static ThaiRomanizationSharp.Thai2Rom.Extensions;
+
+    internal sealed class Seq2Seq : nn.CustomModule
```

Comment:
Could you please give just a short summary of what this class does/for?
Does Seq2Seq stand for `Sequence to Sequence`?